cclient/api/extended/get_inputs.c

```diff
@@ -4,63 +4,130 @@
  *
  * Refer to the LICENSE file for licensing information
  */
-#include "cclient/api/core/get_balances.h"
-
 #include "cclient/api/extended/get_inputs.h"
-#include "cclient/api/extended/get_new_address.h"
+#include "cclient/api/core/find_transactions.h"
+#include "cclient/api/core/get_balances.h"
 #include "cclient/api/extended/logger.h"
+#include "common/helpers/sign.h"
 
-retcode_t iota_client_get_inputs(iota_client_service_t const* const serv, flex_trit_t const* const seed,
-                                 address_opt_t const addr_opt, uint64_t const threshold, inputs_t* const out_inputs) {
+static retcode_t is_unused_address(iota_client_service_t const* const serv, flex_trit_t const* const addr,
```

Comment:
please call `is_unused_address` in get_new_address.c

Reply:
but `is_unused_address` is a static function in `get_new_address.c` which is not able to be included by `get_inputs.c`
That's why I use this ugly solution

Reply:
feel free to make `is_unused_address` public in get_new_address.c